com.unity.multiplayer.mlapi/Tests/Runtime/NetworkVariableTests.cs

```diff
@@ -31,15 +31,15 @@ public void NetworkSerialize(NetworkSerializer serializer)
 public class NetworkVariableTest : NetworkBehaviour
 {
     public readonly NetworkList<int> TheList = new NetworkList<int>(
-        new NetworkVariableSettings { WritePermission = NetworkVariablePermission.ServerOnly }
+        new NetworkVariableSettings {WritePermission = NetworkVariableWritePermission.ServerOnly}
```

Comment:
Automated nitpick: This gives me a `./standards.py` whitespace error. Same for next 2.